tests/album-delete.test.js

```diff
@@ -0,0 +1,42 @@
+const { expect } = require('chai');
+const request = require('supertest');
+const db = require('../src/db');
+const app = require('../src/app');
+
+describe('Delete Album', () => {
+  let album;
+  beforeEach(async () => {
+    const { rows } = await db.query(
```

Comment:
I think it would be a good idea to create an entry in the artist table so that you can send a valid artistId in the SQL query below when you create an album